boards/clue_nrf52840/Makefile

```diff
@@ -10,7 +10,7 @@ ifdef PORT
   FLAGS += -p $(PORT)
 endif
 
-APP=../../../libtock-c/examples/screen/build/cortex-m4/cortex-m4.tbf
+APP=../../../libtock-c/examples/sensors/build/cortex-m4/cortex-m4.tbf
```

Comment:
You should probably avoid updating this variable here?
It's probably best to not hardcode any `APP` variable here at all, but to just take it from the environment as [earlgrey-nexysvideo](https://github.com/tock/tock/blob/b939473ad41f9723605155add18e92cf3d78cd53/boards/earlgrey-nexysvideo/README.md#programming-apps) does.

Reply:
Thanks!